research/2019-12-16_searching_with_the_calm_api.md

```diff
@@ -0,0 +1,241 @@
+# Searching with the CALM API
+
+### 16 December 2019, AWLC
+
+This has some notes on using the CALM API, and an example script for searching all the records in the CALM catalogue.
+
+
+
+## Motivation
+
+As part of the born-digital content migration, I had a collection of directories with accession-like numbers, and I wanted to get a list of all the accession numbers in CALM, so I could match the directories to accession numbers:
```

Comment:
Where did you get the list of directories?

Reply:
From the V drive; I'll clarify in the document.